stable/external-dns/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: external-dns
-version: 2.5.3
+version: 2.5.4
```

Comment:
Since this is a feature rather than a bug fix, semver dictates that the minor version needs to be increased